swarm/network/stream/delivery.go

```diff
@@ -224,21 +215,17 @@ func (d *Delivery) processReceivedChunks() {
 			}
 			// go func() {
```

Comment:
remove comments
 I am not sure if this should be in a go routine at all, if storage cannot keep up with network there is bigger problem here. maybe there should be a just the buffer on receiveC and a select  on line 184 with default dropping the peer or without select 
a select  on line 184 with default dropping the peer or without select just blocking the network protocol until the buffer is freed.